ansible/aws_coreos_site.yml

```diff
@@ -105,6 +148,9 @@
           - protocol: http
             load_balancer_port: 80
             instance_port: 80
+          - protocol: http
+            load_balancer_port: 8080
```

Comment:
Added 8080 to reach Vulcan and aggregator-healthcheck
On 5 Jan 2016 12:03 am, "Catalin Ilea" <notifications@github.com> wrote:

> In ansible/aws_coreos_site.yml
> <https://github.com/Financial-Times/coco-docker-image-cache-provisioner/pull/7#discussion_r48732749>
> :
>
> > @@ -105,6 +148,9 @@
> >            - protocol: http
> >              load_balancer_port: 80
> >              instance_port: 80
> > +          - protocol: http
> > +            load_balancer_port: 8080
>
> Any reason why you changed the elb port to 8080?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/coco-docker-image-cache-provisioner/pull/7/files#r48732749>
> .
>